.github/workflows/release.yml

```
@@ -0,0 +1,377 @@
+  # デプロイメント通知
+  # =============================================================================
+  notify-deployment:
+    name: デプロイメント通知
+    runs-on: ubuntu-latest
+    needs: [create-release, publish-pypi]
+    if: always()
+    
+    steps:
+    - name: 成功通知
+      if: needs.create-release.result == 'success'
+      run: |
```

actionlint (1.7.10)

property "pre-release-validation" is not defined in object type {create-release: {outputs: {}; result: string}; publish-pypi: {outputs: {}; result: string}}

(expression)